### PR TITLE
 Create test for setVisibleListentrieHIDDEN issue#15155

### PR DESCRIPTION
**Pull request by a22albjo**
Create test for setVisibleListentrieHIDDEN issue#15155

### CODE REVIEW

**Review by a22iliru — CHANGES_REQUESTED** [review]
Test 3 (assertEqual): failed
value expected: {"entries":[{"entryname":"HIDDENTEST123","lid":100000,"visible":0}]}
value output: {"entries":[{"entryname":"HIDDENTEST123","visible":0}]}

Test 3 fails, does it need to be fixed? @a22albjo

File: DuggaSys/tests/sectionedService_test.php [hunk 1]
```diff
@@ -415,72 +415,32 @@
...
 
-        'query-after-test-1' => "UPDATE listentries SET visible=1 WHERE lid=2008;",
-    
+    'setVisibleListentrieHIDDEN' => array(
+        'expected-output'   => '{"entries":[{"entryname":"HIDDENTEST123","lid":100000,"visible":0}]}',
```
Comment (a22albjo) [inline DuggaSys/tests/sectionedService_test.php:420]:
```suggestion
        'expected-output'   => '{"entries":[{"entryname":"HIDDENTEST123","visible":0}]}',
```

**Comment by a22albjo** [comment]
> Test 3 (assertEqual): failed value expected: {"entries":[{"entryname":"HIDDENTEST123","lid":100000,"visible":0}]} value output: {"entries":[{"entryname":"HIDDENTEST123","visible":0}]}
> 
> Test 3 fails, does it need to be fixed? @a22albjo

Oops, fixed!

**Review by a22iliru — APPROVED** [review]
good!

**Pull request merged**